.github/workflows/discord-notifications.yml

```
@@ -0,0 +1,30 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR Opened Notification
+        if: github.event.action == 'opened'
+        run: |
+          curl -H "Content-Type: application/json" \
+            -d "{\"embeds\": [{\"title\": \"🆕 New PR Opened\", \"description\": \"**${{ github.event.pull_request.title }}**\", \"url\": \"${{ github.event.pull_request.html_url }}\", \"color\": 3447003, \"fields\": [{\"name\": \"Author\", \"value\": \"${{ github.event.pull_request.user.login }}\", \"inline\": true}, {\"name\": \"Branch\", \"value\": \"${{ github.event.pull_request.head.ref }}\", \"inline\": true}]}]}" \
+            ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+      - name: PR Merged Notification
+        if: github.event.action == 'closed' && github.event.pull_request.merged == true
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)